agent_api/src/main/java/dev/aikido/agent_api/ratelimiting/FixedWindowRateLimiter.java

```diff
@@ -0,0 +1,51 @@
+package dev.aikido.agent_api.ratelimiting;
+
+import static dev.aikido.agent_api.helpers.UnixTimeMS.getUnixTimeMS;
+
+public class FixedWindowRateLimiter implements RateLimiter {
```

Comment:
Do we still need this, if not used? We can always bring it back if needed (reduces diff)

Reply:
Sure